helm/vitess/templates/_vttablet.tpl

```diff
@@ -321,7 +404,9 @@ spec:
         -db-config-filtered-uname "vt_filtered"
         -db-config-filtered-dbname "vt_{{$keyspace.name}}"
         -db-config-filtered-charset "utf8"
+{{ if gt (int $shard.tabletCount) 1 }}
```

Comment:
Can this use the computed `$totalTabletCount`?

Reply:
Yeah, that was an oversight